src/chrome/content/rules/WestMidlandsPolice.xml

```diff
@@ -1,4 +1,4 @@
-<ruleset name="West-midlands.police.uk">
+<ruleset name="West Midlands Police">
   <target host="west-midlands.police.uk" />
   <target host="www.west-midlands.police.uk" />
 
```

Comment:
WFM:
foi.west-midlands.police.uk

mixed content (please check for broken content):
behindthecrime.west-midlands.police.uk
jobs.west-midlands.police.uk
saferstudents.west-midlands.police.uk
safestreet.west-midlands.police.uk